stories/8bit/button-stories.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from "react";
 import { storiesOf } from "@storybook/react";
 
-import { Button } from "../../src/8bit";
+import { Button } from "../../src";
```

Comment:
@andrewgierens similarly to above, the idea here was to import the "8bit button" not just a button. Because we may build many buttons at some point.